### ISSUE TITLE
cursor wont move below a point on screen when zoomed out.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#193]
```
What steps will reproduce the problem?
1. Zoom out
2. Try to teleport, use girders, aim air strike, piano,etc.
3. Zoom in again
4. It works well!

What is the expected output? What do you see instead?
I expect it to allow me to move the cursor to where I want it to be.Instead, I 
see that it does not allow me to move the cursor below a certain point.


What version of the product are you using? On what operating system?
I am using Hedgewars 0.9.15, on Ubuntu 10.04 LTS x86_64.

Please provide any additional information below.


```

-----
Original issue reported on code.google.com by `ross0mcd...@gmail.com` on 26 Feb 2011 at 3:45

**Comment by GoogleCodeExporter** [comment]
```
Known bug.  Problem is related to mouse following cursor and limits of the 
world.
There are a couple of possible discussed solutions.  One is to allow scrolling 
further down when zoomed out (will require adjustin gradient for graphics).  
Another would be to allow the cursor position to move to the edge of the screen 
without moving the camera when it hits the boundary.

```

-----
Original comment by `kyberneticist@gmail.com` on 26 Feb 2011 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Issue 76 has been merged into this issue.
```

-----
Original comment by `vittorio...@gmail.com` on 1 Mar 2011 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed in rev 6533e5450a21
```

-----
Original comment by `unC0Rr` on 2 May 2011 at 2:43
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified, you can move cursor to every place on the map
```

-----
Original comment by `pilarski...@gmail.com` on 29 Nov 2011 at 7:53
* Changed state: **Verified**
